### PR TITLE
Switch to “Activate” wording

### LINKED ISSUE #5922: [v4] Clean up "Register" wording for the license activation
We still have a lot of "Register" wording in the Panel, especially in the registration dialog but also in the system view (license status).

https://github.com/getkirby/kirby/pull/5921#pullrequestreview-1713800963

### PR DESCRIPTION
**Pull request by distantnative**
### Fixes
- #5922

### CODE REVIEW

File: i18n/translations/en.json
Comment (lukasbestle) [inline i18n/translations/en.json]: It's not really the site that gets activated but the license (also makes the status text in the stats card easier to understand/mentally connect):

```suggestion
	"license.activate.label": "Please activate your license",
```

File: i18n/translations/en.json
Comment (lukasbestle) [inline i18n/translations/en.json]:
```suggestion
	"license.activate.local": "You are about to activate your Kirby license for your local domain <strong>{host}</strong>. If this site will be deployed to a public domain, please activate it there instead. If {host} is the domain you want to use your license for, please continue.",
```

File: i18n/translations/en.json
Comment (lukasbestle) [inline i18n/translations/en.json]:
```suggestion
	"license.code.help": "You received your license code after the purchase via email. Please copy and paste it here.",
```

File: i18n/translations/en.json
Comment (lukasbestle) [inline i18n/translations/en.json:425]: Suggestion for an alternate term for the stats card: "Trial mode"

File: i18n/translations/en.json
Comment (distantnative) [reply i18n/translations/en.json]: "the domain you want to license Kirby to" sounds a bit odd.
What about "the domain you want to use your Kirby license for"?

Comment (lukasbestle) [reply i18n/translations/en.json]: Omitted Kirby for brevity (additional line break), otherwise 👍

File: i18n/translations/en.json
Comment (distantnative) [reply i18n/translations/en.json:425]: I could see that - but would be good to get more opinions here. Shall we merge this as is and tackle this when adding also the v4 license wording in (after implementing the backend handling)?

Comment (lukasbestle) [reply i18n/translations/en.json:425]: Either way is fine for me. The other changes in this PR are already a huge improvement. Let's wait for another review and whoever reviews gets to decide whether to change this label or keep it "Unregistered" for now. ;)

Comment (bastianallgeier) [reply i18n/translations/en.json:425]: I like Trial mode. But the wording could also raise questions if there are some deactivated features. Let's merge this already and think about it some more in the next steps as Nico suggested.

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**
